alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/metadataExtractors/OpenDocumentMetadataExtractor.java

```diff
@@ -90,22 +101,33 @@ public OpenDocumentMetadataExtractor()
     @Override
     protected Parser getParser()
     {
-        return new OpenDocumentParser();
+        OpenDocumentParser parser = new OpenDocumentParser();
+        parser.setMetaParser(new OpenDocumentMetaParser() {
+            @Override
+            protected ContentHandler getContentHandler(ContentHandler ch, Metadata md, ParseContext context)
+            {
+                final ContentHandler superHandler = super.getContentHandler(ch, md, context);
+                final ContentHandler creatorHandler = new ElementMetadataHandler(NAMESPACE_URI_DC, KEY_CREATOR, md, KEY_ALFRESCO_CREATOR);
+                return new TeeContentHandler(superHandler, creatorHandler);
+            }
+        });
+        return parser;
     }
 
-    @SuppressWarnings("deprecation")
     @Override
     protected Map<String, Serializable> extractSpecific(Metadata metadata,
                                                         Map<String, Serializable> properties, Map<String, String> headers)
     {
-        putRawValue(KEY_CREATION_DATE, getDateOrNull(metadata.get(Metadata.CREATION_DATE)), properties);
-        putRawValue(KEY_CREATOR, metadata.get(Metadata.CREATOR), properties);
-        putRawValue(KEY_DATE, getDateOrNull(metadata.get(Metadata.DATE)), properties);
-        putRawValue(KEY_DESCRIPTION, metadata.get(Metadata.DESCRIPTION), properties);
+        putRawValue(KEY_CREATION_DATE, getDateOrNull(metadata.get(TikaCoreProperties.CREATED)), properties);
+        final String creator = getCreator(metadata);
+        putRawValue(KEY_CREATOR, creator, properties);
+        putRawValue(KEY_AUTHOR, creator, properties);
+        putRawValue(KEY_DATE, getDateOrNull(metadata.get(TikaCoreProperties.MODIFIED)), properties);
+        putRawValue(KEY_DESCRIPTION, metadata.get(TikaCoreProperties.DESCRIPTION), properties);
         putRawValue(KEY_GENERATOR, metadata.get("generator"), properties);
         putRawValue(KEY_INITIAL_CREATOR, metadata.get("initial-creator"), properties);
-        putRawValue(KEY_KEYWORD, metadata.get(Metadata.KEYWORDS), properties);
-        putRawValue(KEY_LANGUAGE, metadata.get(Metadata.LANGUAGE), properties);
+        putRawValue(KEY_KEYWORD, metadata.get(TikaCoreProperties.SUBJECT), properties);
```

Comment:
Same mapping - is it correct?

Reply:
According to the [migration guide ](https://cwiki.apache.org/confluence/display/TIKA/Migrating+to+Tika+2.0.0) the `KEYWORDS` has been transitioned to `dc:subject`